utils/utils.go

```diff
@@ -332,3 +333,80 @@ func PrintErrorAndExit(message, errorTemplate string) {
 	Tmpl(fmt.Sprintf(errorTemplate, message), nil)
 	os.Exit(2)
 }
+
+// GoCommand executes the passed command using Go tool
+func GoCommand(command string, args ...string) error {
+	allargs := []string{command}
+	allargs = append(allargs, args...)
+	goBuild := exec.Command("go", allargs...)
+	goBuild.Stderr = os.Stderr
+	return goBuild.Run()
+}
+
+// GoBuild runs the "go build" command on the specified package
+func GoBuild(debugname, pkg string) error {
+	buildFlags := "-ldflags='-linkmode internal'"
```

Comment:
Yes you're right. Other packages might use different `ldflags`, but here I am just making sure Go's internal linker is used. I will move this function to `dlv.go`.